velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -17,9 +17,36 @@
 #include "velox/functions/Registerer.h"
 #include "velox/functions/lib/IsNull.h"
 #include "velox/functions/prestosql/Cardinality.h"
+#include "velox/functions/prestosql/GreatestLeast.h"
 #include "velox/functions/prestosql/InPredicate.h"
 
 namespace facebook::velox::functions {
+
+template <typename T>
+inline void registerGreatestLeastFunctionHelper(const std::string& prefix) {
```

Comment:
We don't use names like xxxHelper. Any chance you could rename this function to registerGreatestLeastFunction and the other one to ___s (registerGreatestLeastFunctions)? Or something along these lines. Thanks.

Reply:
How about registerGreatestLeastFunction and registerAllGreatestLeastFunctions ?

Reply:
That works. Thanks.

Reply:
Fixed